### PR TITLE
remove reusable-publish-result; move logic to testing jobs

### PR DESCRIPTION
**Pull request by AndreyMlashkin**

### CODE REVIEW

File: .github/workflows/reusable-submodule-testing-mac.yml [hunk 1]
```diff
@@ -61,6 +61,7 @@ jobs:
       - name: Run checks, copy test results
         run: |
           input="${{ inputs.check-names }}"
+          echo "input is ${input}"
```
Comment (x-mass) [inline .github/workflows/reusable-submodule-testing-mac.yml]: debug print

File: .github/workflows/crypto3-testing-linux.yml [hunk 2]
```diff
@@ -62,12 +62,12 @@ jobs:
             cp -r ./result/* "$results_dir/$check_name"
             rm result
           done
+          ls -la
+          ls -la results/
```
Comment (x-mass) [inline .github/workflows/crypto3-testing-linux.yml]: debug print

**Review by x-mass — APPROVED** [review]

File: .github/workflows/reusable-submodule-testing-mac.yml (hunk 1 above)
Comment (AndreyMlashkin) [reply .github/workflows/reusable-submodule-testing-mac.yml]: fixed

File: .github/workflows/crypto3-testing-linux.yml (hunk 2 above)
Comment (AndreyMlashkin) [reply .github/workflows/crypto3-testing-linux.yml]: fixed

**Pull request merged**